File: Crud Api/Models/Employee.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace Crud.Models
{
    public class Employee 
    {
       [Key]
       public int id { get; set; }

       [Column(TypeName = "nvarchar(500)")]
       public string Name { get; set; }

       [Column(TypeName = "nvarchar(500)")]
        public string Contact { get; set; }


        [Column(TypeName = "nvarchar(500)")]
        public string Email { get; set; }


        [Column(TypeName = "nvarchar(500)")]
        public string DOJ { get; set; }


        [Column(TypeName = "nvarchar(500)")]
        public string Gender { get; set; }


        [Column(TypeName = "nvarchar(500)")]
        public string City { get; set; }


    }
}

File: Crud Api/ClientApp/src/app/models/employee.ts
export class Employee {
  id: number;
  Name: string;
  Contact: number;
  Email: string;
  DOJ: string;
  Gender: string;
  City: string;

  constructor(id: number = null,
    Name: string = '', Contact: number = 0, Email: string = '', DOJ: string='', Gender: string='', City: string='',) {
    this.id = id;
    this.Name = Name;
    this.Contact = Contact;
    this.Email = Email;
    this.DOJ = DOJ;
    this.City = City;
    this.Gender = Gender;

  }
}



